Multisig/UI/Transactions/TransactionDetails/TransactionConfirmationsView/TransactionConfirmationsView.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  ConfirmationsView.swift
+//  Multisig
+//
+//  Created by Moaaz on 6/10/20.
+//  Copyright © 2020 Gnosis Ltd. All rights reserved.
+//
+
+import SwiftUI
+
+struct TransactionConfirmationsView: View {
+    let transaction: BaseTransactionViewModel
+    let safe: Safe
+
+    var body: some View {
+        return VStack(alignment: .leading, spacing: 1) {
+            TransactionConfirmationCell(address: transaction.confirmations!.first!.address, style: .created)
+
+            ForEach(transaction.confirmations ?? [], id: \.address) { confirmation in
```

Comment:
I think better way would be to make the confirmations ad this view's variable and work with that because then, during initializaiton the confirmations can be set as an empty array and no need for this logic or implicit unwrap of the optionals multiple times.

Reply:
the problem here that in some other parts we need the transaction information to create the other parts

Reply:
Now I saw it!